io.github.nokse22.teleprompter.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id" : "io.github.nokse22.teleprompter",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "teleprompter",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home:ro"
```

Comment:
this is a very broad permission. Given that it uses GNOME it should use the portals.